railties/lib/rails/generators/rails/plugin/plugin_generator.rb

```diff
@@ -92,7 +92,7 @@ def test
     ]
 
     def generate_test_dummy(force = false)
-      opts = (options.dup || {}).keep_if {|k, | PASSTHROUGH_OPTIONS.map(&:to_s).include?(k) }
+      opts = (options.dup || {}).keep_if { |k, | PASSTHROUGH_OPTIONS.map(&:to_s).include?(k) }
```

Comment:
Can you also add `_` for unused last arg?
I've surveyed in the code base by `git grep -n '{ |., '`, but looks like it is only place using `|k, |`. we usually use `|k, _|` for the case.